std/traits.d

```diff
@@ -6524,6 +6575,41 @@ unittest
         static assert(mostNegative!T == 0);
 }
 
+/**
+Get the type that a scalar type `T` will $(LINK2 https://dlang.org/spec/type.html#integer-promotions, promote)
```

Comment:
no please don't - see [this](https://github.com/dlang/phobos/pull/4411) for the motivation to kill WEB.